core/gather/gatherers/bf-cache-errors.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import FRGatherer from '../base-gatherer.js';
+import {waitForFrameNavigated, waitForLoadEvent} from '../driver/wait-for-condition.js';
+
+class BFCacheErrors extends FRGatherer {
+  /** @type {LH.Gatherer.GathererMeta} */
+  meta = {
+    supportedModes: ['navigation'],
+  };
+
+  /**
+   * @param {LH.Gatherer.FRTransitionalContext} context
+   * @return {Promise<LH.Artifacts.BFCacheErrors>} All visible tap targets with their positions and sizes
+   */
+  async getArtifact(context) {
+    const session = context.driver.defaultSession;
+
+    /**
+     * @type {LH.Crdp.Page.BackForwardCacheNotRestoredExplanation[]}
+     */
+    let errors = [];
+
+    /**
+     * @param {LH.Crdp.Page.BackForwardCacheNotUsedEvent} event
+     */
+    function onBfCacheNotUsed(event) {
+      errors = event.notRestoredExplanations;
+    }
+
+    session.on('Page.backForwardCacheNotUsed', onBfCacheNotUsed);
+
+    const history = await session.sendCommand('Page.getNavigationHistory');
+    const entry = history.entries[history.currentIndex];
+
+    await Promise.all([
+      session.sendCommand('Page.navigate', {url: 'chrome://terms'}),
```

Comment:
- need to verify this url works in edge and brave
- need to confirm we can't just use about:blank (does the devtools CL suggest why, exactly?)

Reply:
> need to verify this url works in edge and brave

It works but the contents are different for each browser.

> need to confirm we can't just use about:blank (does the devtools CL suggest why, exactly?)

https://source.chromium.org/chromium/chromium/src/+/main:third_party/devtools-frontend/src/front_end/panels/application/components/BackForwardCacheView.ts;l=266-270?q=BackForwardCacheView&ss=chromium%2Fchromium%2Fsrc:third_party%2Fdevtools-frontend%2Fsrc%2F

Seems like `about:blank` would fit the bill as well?

Reply:
Pending response from chrome peeps about if about:blank would suffice here.